packages/shared/styles/components/SfAccordion.scss

```
@@ -0,0 +1,51 @@
+@import '../variables';
+
+$sf-accordion-item__header-padding: 1rem 2rem;
```

```suggestion
$accordion-item__header-padding: 1rem 2rem !default;
```
SCSS vars shouldn't have `$sf-` prefix, and you should use `!default` for all component variables

---

same for all vars below...

---

Thanks, updated :-)